.github/workflows/process-load-test-result.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           while [ ! -z "$status" ]
           do
             sleep 20
-            status=$(gh pr status --jq '.currentBranch .statusCheckRollup[] | select(.status!="COMPLETED") | select(.context!="license/cla")' --json statusCheckRollup)
+            status=$(gh pr status --jq '.currentBranch .statusCheckRollup[] | select(.status!="COMPLETED") | select(.context!="license/cla" | select(.context="codecov/project" )' --json statusCheckRollup)
```

Comment:
This will ignore other status checks. The `|` works like a pipe.
we can do something like this though.
```
checkCount="0"
while [ "$checkCount" != "3" ]
do
  sleep 20
  checkCount=$(gh pr status --jq '[.currentBranch .statusCheckRollup[] | select((.conclusion=="SUCCESS") and ((.name=="Build on Ubuntu") or (.name=="Build on Windows") or (.name=="codecov/project")))]|length' --json statusCheckRollup)
done
```

Reply:
Added in [df7ee1a](https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/150/commits/df7ee1a92a311b353c2e44f49ef74b2427cd0055)